sdk/communication/azure-communication-chat/build.gradle

```diff
@@ -8,10 +8,32 @@ android {
         minSdkVersion 21
         versionCode project.versionCode
         versionName project.version
+
+        def buildVariant = getBuildVariant()
+        def envPropertiesFile
+        if ((buildVariant == "Prod")) {
+            envPropertiesFile = "env-prod.properties"
+        } else if ((buildVariant == "Test")) {
+            envPropertiesFile = "env-test.properties"
+        } else {
+            envPropertiesFile = "env-prod.properties"
+        }
+
+        def envProperties = new Properties()
+        envProperties.load(new FileInputStream("$projectDir/" + envPropertiesFile))
+        println("buildVariant = $buildVariant")
+        for (String key : envProperties.keySet()) {
+            buildConfigField "String", key.replaceAll("\\.", "_").toUpperCase(), envProperties[key]
+        }
     }
 }
 
+static def getBuildVariant() {
+    return "Prod"
+}
```

Comment:
curious, what's the reason for adding this variant? so far we were releasing common and chat without this annotation.

Reply:
It is for switching prod trouter and testing trouter environment. Only we can use testing trouter environment. Users will always use prod trouter.